src/viam/sessions_client.py

```diff
@@ -44,12 +44,23 @@ class SessionsClient:
     supports stopping actuating components when it's not.
     """
 
-    def __init__(self, channel: Channel, address: str, dial_options: Optional[DialOptions], *, disabled: bool = False):
+    def __init__(self, channel: Channel, direct_dial_address: str, dial_options: Optional[DialOptions], *, disabled: bool = False):
         self.channel = channel
         self.client = RobotServiceStub(channel)
-        self._address = address
+        self._address = direct_dial_address
         self._dial_options = dial_options
         self._disabled = disabled
+        if dial_options is not None:
+            self._dial_options = DialOptions(
+                disable_webrtc=True,
+                auth_entity=dial_options.auth_entity,
+                credentials=dial_options.credentials,
+                insecure=dial_options.insecure,
+                allow_insecure_downgrade=dial_options.allow_insecure_downgrade,
+                allow_insecure_with_creds_downgrade=dial_options.allow_insecure_with_creds_downgrade,
+            )
```

Comment:
Let's use deepcopy here (`from copy import deepcopy`), otherwise we'll need to remember to add new dial option fields here too.

```suggestion
        if dial_options is not None:
            self._dial_options = deepcopy(dial_options)
```

Reply:
done